source/docs/yearly-overview/known-issues.rst

```diff
@@ -49,8 +49,12 @@ There is currently an issue with inter-operating C++/Java Network Tables clients
 
 **Workaround**: Write code on the server to mirror any keys you wish to see on other clients (e.g. dashboards) to a separate key. For example, if you have a key named ``targetX`` being published by a co-processor that you want to show up on a dashboard, you could write code on the robot to read the key and re-write it to a key like ``targetXDash``.
 
-Resolved Issues
----------------
+Fixed in Game Tools 2021 f1
```

Comment:
f1 isn't clear, can this be reworded?

Reply:
Do you have suggestions? Game Tools is the name of the tool and 2021 f1 is the version. 

![image](https://user-images.githubusercontent.com/4885091/104850914-b6d28f00-58a6-11eb-8973-39c73f3ee490.png)

Reply:
Didn't realize it was versioned